src/CONSTANTS/constants.js

```diff
@@ -6,8 +6,8 @@
  * @description constants
  */
 
-//const mongoDBUrl='mongodb://skunk:skunkcovid19@54.226.1.143:27017/CVD19DEV'
-const mongoDBUrl = 'mongodb://localhost:27017/CVD19DEV';
+//const mongoDBUrl = 'mongodb://localhost:27017/CVD19DEV';
+const mongoDBUrl='mongodb://Skunks:Skunks2019@covid-mongo.cluster-cpzbdytelbqi.us-east-1.docdb.amazonaws.com:27017/CVD19DEV?ssl=true&ssl_ca_certs=rds-combined-ca-bundle.pem&replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false'
```

Comment:
I'm not too familiar with DocumentDB, but if these are the db user and password, we probably want to use environment variables instead? Like:
```suggestion
const mongoDBUrl=`mongodb://${process.env.DB_USER}:${process.env.DB_USER_PASSWORD}@${process.env.DB_CLUSTER}.us-east-1.docdb.amazonaws.com:27017/CVD19DEV?ssl=true&ssl_ca_certs=rds-combined-ca-bundle.pem&replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false`
```
We'll want new credentials, as well, if that is the case.